awx/ui/src/screens/Inventory/InventorySources/InventorySourceList.js

```diff
@@ -50,12 +57,20 @@ function InventorySourceList() {
         sourceCount: results[0].data.count,
         sourceChoices: results[1].data.actions.GET.source.choices,
         sourceChoicesOptions: results[1].data.actions,
+        searchableKeys: Object.keys(results[1].data.actions?.GET || {}).filter(
+          (key) => results[1].data.actions?.GET[key].filterable
+        ),
+        relatedSearchableKeys: (
+          results[1]?.data?.related_search_fields || []
+        ).map((val) => val.slice(0, -8)),
```

Comment:
Do we need to verify `__search` here?

Reply:
No, not here.  It seems to be that only the `ansible_facts` related search key does not end in `__search`, which I could argue really is an API issue.  If/when it's fixed the [this line](https://github.com/ansible/awx/pull/10959/files#diff-18074f2fc3330129129451ed5e744021a2339f8c15b2e0afc624871b489518cbR71) won't be needed.  AFAIK `ansible_facts` only shows up in lists that have to do with hosts.